notebooks/getting_started/README.md

```diff
@@ -0,0 +1,50 @@
+# VRS Python Getting Started Notebook Series
+This series of notebooks is intended to be the fastest way to becoming productive with the 
+[vrs-python](https://github.com/ga4gh/vrs-python) package. The 
+intent of each notebook in this series is to be an interactive introduction to functionally contained in the vrs-python package.
+
+A beginning developer level of familiarity with python, jupyter notebooks is assumed in order to run this notebook series. 
+You should be familiar with installing packages and running commands in your execution environment.
+
+### Pre-requisites
+The following software packages must exist in your execution environment before running these notebooks:
+* git
+* postgresql@14
```

Comment:
The nice thing about these notebooks is that you don't even need postgresql to run! :) 
```suggestion
```

Reply:
@korikuzma good candidate for a separate issue imho